services/web/server/src/simcore_service_webserver/projects/models.py

```diff
@@ -95,12 +96,36 @@ class UserProjectAccessRightsWithWorkspace(BaseModel):
 
 class ProjectPatchExtended(ProjectPatch):
     # Only used internally
-    trashed_at: datetime | None
+    trashed_at: dt.datetime | None
     trashed_explicitly: bool
 
     model_config = ConfigDict(populate_by_name=True, extra="forbid")
 
 
+class NodeDB(BaseModel):
+    node_id: NodeID
+    required_resources: dict[str, Any]
+    created: dt.datetime
+    modified: dt.datetime
+    project_uuid: ProjectID
+    project_node_id: int
+    key: str
+    version: str
+    label: str
+    progress: float | None
+    thumbnail: HttpUrl | None
+    input_access: dict[str, Any]
+    input_nodes: list[NodeID]
+    inputs: dict[str, Any]
+    inputs_units: dict[str, Any]
+    output_nodes: list[NodeID]
+    outputs: dict[str, Any]
+    run_hash: str | None
+    state: dict[str, Any] | None
+    parent: str | None
+    boot_options: dict[str, Any] | None
```

Comment:
so this is not a duplicate of what is in utils_project_nodes.py without all the default values? I guess we see again the problem that postgres-database does not handle basic pydantic models. ok.

Reply:
Yes, that was just temporary